python/lsst/meas/extensions/gaap/_gaap.py

```diff
@@ -517,7 +528,7 @@ def _gaussianizeAndMeasure(self, measRecord: lsst.afw.table.SourceRecord,
                 baseName = self.ConfigClass._getGaapResultName(scalingFactor, sigma, self.name)
                 if sigma <= targetSigma:
                     # Raise when the aperture is invalid
-                    self._setFlag(measRecord, baseName)
+                    self._setFlag(measRecord, baseName, "bigpsf")
```

Comment:
Should this flag suffix be either `camelCase` (`bigPsf`) or `snake_case` (`big_psf`) rather than all in one? I realize that this flag appears elsewhere in this script (and therefore perhaps perhaps outside the scope of this ticket), but it caught my eye, so I wanted to mention it.

Reply:
Thanks Lee. I think it should indeed be `bigPsf`, similar to flags like `*_flag_badCentroid`, `*_flag_noInputs` elsewhere in `meas_base`. I'll add a separate commit that changes this throughout the package!